flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -926,7 +926,7 @@ private TableResultInternal executeQueryOperation(QueryOperation operation) {
                                     DataTypeUtils.expandCompositeTypeToSchema(
                                             sinkOperation.getConsumedDataType()),
                                     resultProvider.getRowDataStringConverter(),
-                                    PrintStyle.DEFAULT_MAX_COLUMN_WIDTH,
```

Comment:
Do you mean removing `PrintStyle.DEFAULT_MAX_COLUMN_WIDTH` completely? How to make it configurable? Could you elaborate on it?

Reply:
If we introduce this option, I think `PrintStyle.DEFAULT_MAX_COLUMN_WIDTH` is useless. When building `TableauStyle`, we should read from TableConfig to get the value for max column width.

Reply:
Since sql-client.display.max-column-width only used for `ExecMode.STREAMING`, this PR is only focusing on `ExecMode.STREAMING`. `PrintStyle.DEFAULT_MAX_COLUMN_WIDTH` will still be used for `ExecMode.BATCH`

Reply:
It will be removed while removing deprecated sql-client.display.max-column-width. A follow-up ticket has been created: https://issues.apache.org/jira/browse/FLINK-30758